.github/workflows/ci.yml

```diff
@@ -82,20 +85,21 @@ jobs:
             with:
                 python-version: ${{ matrix.python-version }}
         -   name: "Main Script"
+            env:
+                OS_NAME: ${{ matrix.os }}
             run: |
-                # not expected to run without matplotlib and vtk
-                rm examples/vtk-sample-elements.py
-
-                if [[ $(uname) == "Linux" ]]; then
+                EXTRA_INSTALL="numpy pybind11 meson-python ninja"
+                PROJECT_INSTALL_FLAGS="--no-build-isolation"
+                if [[ "$OS_NAME" == ubuntu-* ]]; then
                     ./.ci/install-silo-on-github.sh
-                    USE_SILO="--use-silo"
+                    PROJECT_INSTALL_FLAGS+=" --config-settings setup-args=-Duse-silo=true"
                 fi
-                EXTRA_INSTALL="numpy pybind11"
-                curl -L -O https://gitlab.tiker.net/inducer/ci-support/raw/main/ci-support.sh
-                . ./ci-support.sh
-                ./configure.py $USE_SILO
+
+                curl -L -O https://tiker.net/ci-support-v0
+                . ci-support-v0
+
                 build_py_project_in_venv
-                run_examples
+                run_examples --no-require-main
```

Comment:
Hm, I don't think the examples were running before. None of them have a `__main__` except `vtk-sample-elements.py` and that one was deleted before. Should be back now!

Reply:
There seem to be some failures once in a while:
https://github.com/inducer/pyvisfile/actions/runs/9873441502/job/27265701017
No idea what that might have been about..